checker-qual/src/main/java/org/checkerframework/checker/mustcall/qual/MustCallAlias.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This polymorphic annotation represents an either-or must-call obligation. This annotation should
+ * always be used in pairs: one on a method receiver parameter or constructor parameter, and the
+ * other on the method return type. Fulfilling the must-call obligation of one is equivalent to
+ * fulfilling the must-call obligation of the other.
+ *
```

Comment:
Show a code example where the annotation should be used.  Then, the below can be shorter (for example, the second, and likely the third, sentence can be removed).